magefiles/magefile.go

```diff
@@ -423,7 +423,7 @@ type Docs mg.Namespace
 // Serve launches MkDocs development server to preview the documentation page
 func (Docs) Serve() error {
 	const (
-		mkdocsImage = "aquasec/mkdocs-material:dev"
+		mkdocsImage = "squidfunk/mkdocs-material:9.5"
```

Comment:
Does this image work for you? I get the following when I try to locally serve:

```
mage docs:serve
[+] Building 0.7s (8/8) FINISHED                                                                       docker:desktop-linux
 => [internal] load build definition from Dockerfile                                                                   0.0s
 => => transferring dockerfile: 280B                                                                                   0.0s
 => [internal] load .dockerignore                                                                                      0.0s
 => => transferring context: 2B                                                                                        0.0s
 => [internal] load metadata for docker.io/squidfunk/mkdocs-material:9.5.44                                            0.6s
 => [internal] load build context                                                                                      0.0s
 => => transferring context: 75B                                                                                       0.0s
 => [1/3] FROM docker.io/squidfunk/mkdocs-material:9.5.44@sha256:ce587cbffd5283056df4a84bd3f2eb0c54f0031b1789844dcaf6  0.0s
 => CACHED [2/3] COPY requirements.txt .                                                                               0.0s
 => CACHED [3/3] RUN pip install -r requirements.txt                                                                   0.0s
 => exporting to image                                                                                                 0.0s
 => => exporting layers                                                                                                0.0s
 => => writing image sha256:af19af90aff1980a44aa0440c744be02ddeebd7cf2e5a735f3c4fb0140c84f73                           0.0s
 => => naming to docker.io/squidfunk/mkdocs-material:9.5.44                                                            0.0s
INFO     -  Building documentation...
Traceback (most recent call last):
  File "/usr/local/bin/mkdocs", line 8, in <module>
    sys.exit(cli())
             ^^^^^
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/mkdocs/__main__.py", line 181, in serve_command
    serve.serve(dev_addr=dev_addr, livereload=livereload, watch=watch, **kwargs)
  File "/usr/local/lib/python3.11/site-packages/mkdocs/commands/serve.py", line 63, in serve
    config = builder()
             ^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/mkdocs/commands/serve.py", line 37, in builder
    config = load_config(
             ^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/mkdocs/config/base.py", line 224, in load_config
    errors, warnings = cfg.validate()
                       ^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/mkdocs/config/base.py", line 116, in validate
    post_failed, post_warnings = self._post_validate()
                                 ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/mkdocs/config/base.py", line 96, in _post_validate
    config_option.post_validation(self, key_name=key)
  File "/usr/local/lib/python3.11/site-packages/mkdocs/config/config_options.py", line 528, in post_validation
    config[key_name] = theme.Theme(**theme_config)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/mkdocs/theme.py", line 45, in __init__
    self._load_theme_config(name)
  File "/usr/local/lib/python3.11/site-packages/mkdocs/theme.py", line 78, in _load_theme_config
    theme_dir = utils.get_theme_dir(name)
                ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/mkdocs/utils/__init__.py", line 317, in get_theme_dir
    return os.path.dirname(os.path.abspath(theme.load().__file__))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen posixpath>", line 399, in abspath
TypeError: expected str, bytes or os.PathLike object, not NoneType
Error: running "docker run --name mkdocs-serve --rm -v /Users/repos/trivy:/docs -p 8000:8000 squidfunk/mkdocs-material:9.5" failed with exit code 1
```

Reply:
This commit https://github.com/aquasecurity/trivy/pull/7884/commits/2fcd0aed95e0eddfbff6fed74d5be31c9a66df17 causes the error.

Reply:
fixed